### PR TITLE
updating for various environments

### PR DESCRIPTION
**Pull request by matt-raffel-kiva**
Signed-off-by: Matt Raffel <mattr@kiva.org>

### CODE REVIEW

File: src/stats/stats.service.ts [hunk 1]
```diff
@@ -58,6 +84,7 @@ export class StatsService {
         };
         const result = await this.http.requestWithRetry(req);
         Logger.info(`${serviceName} returned `, result.data);
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-return
```
Comment (natesulat) [inline src/stats/stats.service.ts]: Nit: We disabled this globally in various other repos, so I think we could do it here, too

**Review by natesulat — COMMENTED** [review]
Why remove `npm run test:local`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -92,7 +91,7 @@ workflows:
               ignore: /.*/
           update-description: true
           dockerfile: Dockerfile.production
-          image: kivaprotocol/protocol-reporting
+          image: kivaprotocol/protocol-system-services
           tag: << pipeline.git.tag >>,latest
       - push-to-dev:
```
Comment (ghost) [inline .circleci/config.yml:97]: Need to update the `push-to-dev` job to push the proper image to dev. Right now it's configured to push protocol-reporting.

**Comment by matt-raffel-kiva** [comment]
> Why remove `npm run test:local`?

removed it because we don't use that consistently across all repos.  I can add it back.

**Review by ghost — CHANGES_REQUESTED** [review]
Just the one issue with the push-to-dev job.

File: .circleci/config.yml (hunk 2 above)
Comment (matt-raffel-kiva) [reply .circleci/config.yml:97]: what does this do:  image: kivaprotocol/protocol-system-services

Comment (ghost) [reply .circleci/config.yml:97]: That is for publishing the docker image to docker hub. Separately, we need to push the image to our dev kubernetes cluster. Check out the job described on line 51 - you'll see what you need to update.

Comment (matt-raffel-kiva) [reply .circleci/config.yml:97]: take a look at the changes.  let me know what else needs updating

**Review by mac-arrap — APPROVED** [review]
merge when @jeffk-kiva concerns are met

File: src/stats/stats.service.ts (hunk 1 above)
Comment (matt-raffel-kiva) [reply src/stats/stats.service.ts]: updated rc file, removed comment

**Review by natesulat — APPROVED** [review]
I'd wait for @jeffk-kiva , too, but LGTM

**Review by ghost — APPROVED** [review]
Looks good. Just make sure you've created the repository in Docker Hub before you tag the first version.

**Pull request merged**